src/components/expansion-panel/expansion-panel/expansion-panel.scss

```diff
@@ -3,19 +3,19 @@
 // Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
 @include sbb.box-sizing;
 
+// Open/Close animation vars
+$open-anim-rows-from: 0fr;
+$open-anim-rows-to: 1fr;
+$open-anim-opacity-from: 0;
+$open-anim-opacity-to: 1;
+
```

Comment:
I've noticed this pattern (with the same variable names and animations) is repeated in 4 components, would it make sense to extract it in a mixin so that we can reuse it in a more efficient way?

Reply:
I've thought about creating a mixin for the open/close animation that would also include the declaration. However, even if part of it is in common, the resulting mixin would have too many parameters. 
I feel like factorizing only a part of the implementation is a bad practice :/